scripts/create_newcase

```diff
@@ -168,11 +164,19 @@ OR
     if args.input_dir is not None:
         args.input_dir = os.path.abspath(args.input_dir)
 
+    if args.ninst_couplers:
+        ncouplers = args.ninst
+        ninst = 1
```

Comment:
I am sure - because this is implemented over the existing multiinstance code some clever relationships had to be established - primarily if NINST_CPL>1 the NINST_* for all other components must == 1 and the entire model (coupler + components) is replicated NINST_CPL times.   If NINST_* > 1 for any component other than coupler then those components are replicated within  the single coupler.   There are reasons that we want to retain the old behavior SPCAM and any case in which we want multiple instances of one component but not others will need to use the old NINST behavior.